CoreScriptsRoot/CoreScripts/GamepadMenu.lua

```diff
@@ -581,6 +580,9 @@ local function setupGamepadControls()
 			gamepadSettingsFrame:TweenSizeAndPosition(UDim2.new(0,102,0,102), UDim2.new(0.5,-51,0.5,-51),
 														Enum.EasingDirection.Out, Enum.EasingStyle.Sine, 0.1, true, 
 				function()
+					if not InputService.VREnabled then
+						pcall(function() InputService.OverrideMouseIconBehavior = Enum.OverrideMouseIconBehavior.None end)
```

Comment:
This is what the old code was doing, but I moved it to make sure the cursor doesn't show for a single frame in VR. The setOverrideMouseIconBehavior() function only shows/hides it, but doesn't reset to none.